src/antsibull_core/subprocess_util.py

```diff
@@ -47,6 +48,7 @@ async def async_log_run(
     stdout_loglevel: str | None = None,
     stderr_loglevel: str | None = 'debug',
     check: bool = True,
+    decode_error_handling: str = 'surrogateescape',
```

Comment:
```suggestion
    *,
    errors: str = 'surrogateescape',
```

I'd make this a keyword only argument and name it like subprocess.run does. I'd also prefer if this was explicitly added to `log_run()` and the `VenvRunner` methods' signatures instead of relying on `**kwargs`.

Reply:
Ok. But shouldn't `logger`, `stdout_loglevel`, `stderr_loglevel`, and `check` also be keyword only arguments?